lighthouse-core/gather/gather-runner.js

```diff
@@ -699,6 +699,57 @@ class GatherRunner {
     }
   }
 
+  /**
+   * @param {Array<LH.Config.Pass>} passConfigs
+   * @param {{driver: Driver, requestedUrl: string, settings: LH.Config.Settings}} options
+   * @return {Promise<LH.Artifacts>}
+   */
+  static async _runWithoutPageLoadForDevtoolsA11y(passConfigs, options) {
+    const driver = options.driver;
+
+    /** @type {Partial<LH.GathererArtifacts>} */
+    const artifacts = {};
+
+    const baseArtifacts = await GatherRunner.initializeBaseArtifacts(options);
+
+    try {
+      await driver.connect();
+
+      await GatherRunner.setupDriver(driver, options);
+
+      for (const passConfig of passConfigs) {
```

Comment:
seems like we could even drop this loop and just create an explicit pass config that only has the a11y gatherer